packages/sit-onyx/src/components/OnyxDataGrid/OnyxDataGridRenderer/OnyxDataGridRenderer.vue

```diff
@@ -3,7 +3,10 @@ import OnyxTable from "../../OnyxTable/OnyxTable.vue";
 import type { DataGridEntry, DataGridMetadata } from "../types";
 import type { OnyxDataGridRendererProps } from "./types";
 
-const props = defineProps<OnyxDataGridRendererProps<TEntry, TMetadata>>();
+const props = withDefaults(defineProps<OnyxDataGridRendererProps<TEntry, TMetadata>>(), {
+  striped: true,
```

Comment:
Usually this would conflict with our guidelines, but here it makes sense. You might want to add a comment about it :)